src/frontend/screens/Onboarding/OnboardingPrivacyPolicy.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react';
+import {
+  View,
+  Text,
+  ScrollView,
+  StyleSheet,
+  TouchableOpacity,
+} from 'react-native';
+import {useIntl, defineMessages} from 'react-intl';
+import {PrivacyPolicy} from '../PrivacyPolicy';
+import {BLUE_GREY, WHITE, BLACK, COMAPEO_BLUE} from '../../lib/styles';
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons';
+
+import {AppDiagnosticMetrics} from '../../metrics/AppDiagnosticMetrics';
+import {DeviceDiagnosticMetrics} from '../../metrics/DeviceDiagnosticMetrics';
+
+const m = defineMessages({
+  navTitle: {
+    id: 'screens.OnboardingPrivacyPolicy.navTitle',
+    defaultMessage: 'Privacy Policy',
+  },
+  permissionsTitle: {
+    id: 'screens.OnboardingPrivacyPolicy.permissionsTitle',
+    defaultMessage: 'Current Permissions',
+  },
+  shareDiagnostics: {
+    id: 'screens.OnboardingPrivacyPolicy.shareDiagnostics',
+    defaultMessage: 'Share Diagnostic Information',
+  },
+});
+
+export const OnboardingPrivacyPolicy = () => {
+  const appDiagnosticMetrics = new AppDiagnosticMetrics();
+  const deviceDiagnosticMetrics = new DeviceDiagnosticMetrics();
+  const {formatMessage} = useIntl();
+  const [isDiagnosticsEnabled, setIsDiagnosticsEnabled] = React.useState(false);
+
+  const toggleDiagnostics = () => {
+    setIsDiagnosticsEnabled(prev => !prev);
+  };
+
+  React.useEffect(() => {
+    appDiagnosticMetrics.setEnabled(isDiagnosticsEnabled);
+    deviceDiagnosticMetrics.setEnabled(isDiagnosticsEnabled);
+  }, [isDiagnosticsEnabled]);
+
+  return (
+    <ScrollView contentContainerStyle={styles.scrollContent}>
+      <PrivacyPolicy />
+      <View style={styles.horizontalLine} />
+      <Text style={styles.header}>{formatMessage(m.permissionsTitle)}</Text>
+      <View style={styles.permissionBox}>
+        <Text style={styles.permissionText}>
+          {formatMessage(m.shareDiagnostics)}
+        </Text>
+        <TouchableOpacity
+          onPress={toggleDiagnostics}
+          style={[
+            styles.checkBox,
+            isDiagnosticsEnabled && styles.checkBoxChecked,
+          ]}>
+          {isDiagnosticsEnabled && (
+            <MaterialIcons name="check" size={18} color={WHITE} />
+          )}
+        </TouchableOpacity>
+      </View>
+    </ScrollView>
+  );
+};
+
+OnboardingPrivacyPolicy.navTitle = m.navTitle;
+
+const styles = StyleSheet.create({
+  scrollContent: {
+    backgroundColor: 'white',
+    padding: 20,
+    marginHorizontal: 10,
```

Comment:
nit: generally avoid using `margin` (especially horizontal ones) as they can occasionally result in overflow not being detected properly. here's a somewhat extreme example since i'm using a rather large system font size to demonstrate this, but it could also happen on narrower devices:

<img width="200" alt="image" src="https://github.com/user-attachments/assets/db1f130d-3edb-4cab-bede-eeb76cb9d338">

My typical recommendation is to almost always avoid margin and instead use container elements to group spaced items and specify the `gap` styling

Reply:
> My typical recommendation is to almost always avoid margin and instead use container elements to group spaced items and specify the gap styling

Sorry, might've been confusing to see this in the context of the line that this commenting on. I should've mentioned this part more at the top-level or in more relevant parts of the PR